src/wallet/rpcwallet.cpp

```diff
@@ -2433,8 +2433,8 @@ UniValue z_listunspent(const UniValue& params, bool fHelp)
             "Optionally filter to only include notes sent to specified addresses.\n"
             "When minconf is 0, unspent notes with zero confirmations are returned, even though they are not immediately spendable.\n"
             "Results are an array of Objects, each of which has:\n"
-            "{txid, type, jsindex, jsoutindex, confirmations, address, amount, memo} (Sprout)\n"
-            "{txid, type, outindex, confirmations, address, amount, memo} (Sapling)\n"
+            "{txid, pool, jsindex, jsoutindex, confirmations, address, amount, memo} (Sprout)\n"
+            "{txid, pool, outindex, confirmations, address, amount, memo} (Sapling)\n"
```

Comment:
Can the output include Orchard notes?
(If not and there is no issue tracking this, please file one and mark it as blocking 5.0.0.)

Reply:
Ah that is #5683.